modules/openapi-generator/src/main/resources/rust-mayastor/tower-hyper/client/configuration.mustache

```diff
@@ -336,9 +336,9 @@ impl Configuration {
         let client = {
             match certificate {
                 None => {
-                    let mut http = hyper_tls::HttpsConnector::new();
+                    let mut http = hyper::client::HttpConnector::new();
```

Comment:
Haven't looked at this connections for a while so pardon my lack of understanding here, but what is the difference here, using the HttpConnector?